cms/templates/molecules.html

```diff
@@ -0,0 +1,121 @@
+{% import 'atoms.html' as atoms %}
+
+{% macro list_of_languages(view, languages) -%}
+<ul>
+{% for lang_code, lang_name in languages %}
+  <li>
+    <a href={{view.request.route_url('locale_matched',  language=lang_code ) }}>{{lang_name}}</a>
+  </li>
+{% endfor %}
+	<li><a href="/spice/locale/change/">&gt;&gt;</a></li>
+</ul>
+{%- endmacro %}
+
+{% macro banner_with_image(src) -%}
+<div class ="banner">
+	{{ atoms.image(src, "", "")}}
```

Comment:
@nathanbegbie we can support bigger images. We're currently using `300xh` for the homepage. We can specifiy bigger images using `view.get_image_url(localisation.image_host, localisation.image, 640, 0)`. Please allow for `srcset` with the default being the same as the `src`